pkg/operator/desiredstatehash/builder.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2020 PlanetScale Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package desiredstatehash can be used to build up an annotation value that
```

Comment:
Also, one comment: Should this change the usage of the `AnnotationsHash` as well?

Reply:
I think I replaced all previous examples of this pattern with the new, combined annotation. Did I miss anything?